Formula/zig.rb

```diff
@@ -27,7 +27,15 @@ class Zig < Formula
   fails_with gcc: "5" # LLVM is built with GCC
 
   def install
-    system "cmake", "-S", ".", "-B", "build", "-DZIG_STATIC_LLVM=ON", *std_cmake_args
+    cpu = case Hardware.oldest_cpu
+    when :arm_vortex_tempest then "apple_m1" # See `zig targets`.
+    else Hardware.oldest_cpu
+    end
```

Comment:
Trying our CPU target from `ncdu`. Alternatively, could revert to 0.10.0 and older build default of `baseline`. I assume `baseline` is a less optimized / more portable option but haven't found much documentation on it. Upstream CI scripts do use `baseline`, e.g. https://github.com/ziglang/zig/blob/master/ci/x86_64-linux-release.sh#L10